src/Quadrature.jl

```diff
@@ -89,7 +89,8 @@ function transform_inf(t , p , f , lb , ub)
 	function v(t)
 		return t.*_none + v_inf(t).*_inf + v_semiinf(t , lb , 1).*semiup + v_semiinf(t , ub , 0).*semilw
 	end
-	j = det(ForwardDiff.jacobian(x ->v(x), t))
+    jac = Zygote.@ignore ForwardDiff.jacobian(x ->v(x), t)
```

Comment:
is that correct?

Reply:
The MWE outputs a correct answer at least

<img width="770" alt="Screenshot 2021-09-27 at 10 02 55" src="https://user-images.githubusercontent.com/26853713/134868391-a8a3da51-8b57-4014-b15a-8b0f80341b6c.png">

Reply:
Oh I guess it only comes into play when differentiating w.r.t. t?

Reply:
Yeah I think so. It was basically the only way I got it to work around the array mutation restriction.